unit/analyses/ai/ai.cpp

```diff
@@ -12,7 +12,7 @@ Author: Diffblue Ltd.
 #include <testing-utils/message.h>
 #include <testing-utils/use_catch.h>
 
-#include <analyses/ai.h>
+#include <util/ai.h>
```

Comment:
the header file and cpp file shouldn't be in separate modules and I think ai should stay in analyses